File: C/1020.c
#include <stdio.h>

int main(void) {
  int diasV;
  scanf("%d", &diasV);

  int ano; 
  int mes; 
  int dia;

  int restoAno;
  int restoMes;

  ano = diasV/365;
  restoAno = diasV%365;

  mes = restoAno/30;
  restoMes = restoAno%30;

  printf("%d ano(s)\n", ano);
  printf("%d mes(es)\n", mes);
  printf("%d dia(s)\n", restoMes);
  return 0;
}

File: Python/1020.py
total = int(input())

anos = int(total / 365)
total = int(total % 365)

meses = int(total / 30)
total = int(total % 30)

dias = total

print('{} ano(s)' .format(anos))
print('{} mes(es)' .format(meses))
print('{} dia(s)' .format(dias))